README.md

```diff
@@ -19,4 +27,7 @@
 - Make a pull request on github.
 - Submit the link to your pull request on Canvas.
 
-[Additional instructions]
+## Bonus
```

Comment:
Bonus looks great! Can you add a `spec` line to the circle config so the bonus test file does not run on Circle?